package.json

```diff
@@ -38,16 +38,17 @@
     "@storybook/react": "^7.6.17",
     "@storybook/react-webpack5": "^7.6.17",
     "@storybook/test": "^7.6.17",
-    "@testing-library/cypress": "^10.0.2",
-    "@testing-library/jest-dom": "^6.4.5",
-    "@testing-library/react": "^15.0.7",
-    "@testing-library/user-event": "^14.5.2",
+    "@testing-library/cypress": "^7.0.6",
+    "@testing-library/jest-dom": "^5.16.4",
+    "@testing-library/react": "^13.0.1",
```

Comment:
If you upgrade to 16, `@testing-library/dom` becomes a peer dep avoiding duplicate installs of it that may have required this to be rolled back.

You really don't want to stay on 13 for React 19 since that would flood your console with deprecation warnings.